.github/workflows/tests.yml

```diff
@@ -33,12 +36,13 @@ jobs:
               run: python3 -m build
 
             - name: Install AlphaPeel
-              run: pip install dist/*.whl
+              run: pip install dist/AlphaPeel-1.1.3-py3-none-any.whl
```

Comment:
Can we make this version agnostic?

Reply:
@gregorgorjanc The wildcard works fine on MacOS and Ubuntu but not on Windows. I couldn't figure out why so I just hardcoded the file name.

Reply:
Let’s worry about this later then